apps/native/app/src/stores/offline-store.ts

```diff
@@ -0,0 +1,52 @@
+import createUse from 'zustand'
+import create from 'zustand/vanilla'
+import { isDefined } from '../utils/is-defined'
+
+interface State {
```

Comment:
Most of the other stores are extending `State` from `zustand/vanilla`, any reason you are not doing that here?

Reply:
But I think I will name it OfflineState and OfflineActions